source/sitemapindex.xml.erb

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```

Comment:
Technically this file will have no replacement.  Did Adriana pass this by Jaywing for review?

Reply:
[Response in this ticket](https://app.asana.com/0/1202529446124674/1202557665702228)